docs/schemaform/form-config.md

```diff
@@ -140,7 +140,19 @@ This does not apply to array fields; for those, you still need to specify an `it
 ```
 
 ### uiSchema configuration
-In addition to the uiSchema options listed in the library [https://github.com/mozilla-services/react-jsonschema-form#the-uischema-object](docs), we have some additional options that are supported for all forms:
+If you're not already familiar with the rjsf uiSchema options, check out the [https://github.com/mozilla-services/react-jsonschema-form#the-uischema-object](library docs). Some commonly used options include:
```

Comment:
Blasted...I was only looking at the `[]` and `()`--which are in the right order--but it's the content inside. The text goes first and link goes second. There it is.